data/json/items/armor.json

```diff
@@ -4014,7 +4014,15 @@
     "warmth": 10,
     "material_thickness": 3,
     "flags": [ "BELTED", "WATER_FRIENDLY" ],
-    "use_action": "DISASSEMBLE"
+    "use_action": "GOBAG"
+  },
+  {
+    "id": "personal_gobag",
+    "type": "ARMOR",
+    "name": "personal go bag",
+    "description": "A huge duffel bag with backpack attached, both packed to the gills.  You have packed this to be ready for deployment.  Activate to unpack and enjoy.",
+    "volume": 144,
```

Comment:
Is there any reason for it to be larger than a normal go bag?

Reply:
I'm using volume as a way to determine if the player used a normal or personal go bag since I using the item's id is fragile.

Reply:
Using volume for that is much more fragile, I think. If someone decides to change go bag's volume because it's too big or too small, it'll be broken. With IDs, people at least try to keep them unchanged to not ruin save compatibility.

Reply:
I'd strongly recommend using a flag or maybe field instead of and id or the volume. Both those solutions are fragile, and in my opinion, checking for a specific ID is bad practice.

Reply:
I agree, I took another approach by using the way the alcohol methods are handled. It works better in my opinion.